src/SFML/Audio/SoundFileReaderOgg.hpp

```diff
@@ -114,8 +114,8 @@ class SoundFileReaderOgg : public SoundFileReader
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    OggVorbis_File m_vorbis;       // ogg/vorbis file handle
-    unsigned int   m_channelCount; // number of channels of the open sound file
+    OggVorbis_File m_vorbis;          // ogg/vorbis file handle
```

Comment:
Yes you're right. When I wrote this commit months ago, I didn't realize that seeing `m_someMember()` in an initializer list could be significant. That's a no-op for types with default constructors but it's critical for types from C interfaces so that they're initialized with zeroes.